api/src/main/java/com/github/eyefloaters/console/api/errors/client/AbstractNotFoundExceptionHandler.java

```diff
@@ -13,7 +13,7 @@
 
 abstract class AbstractNotFoundExceptionHandler<T extends Throwable> implements ExceptionMapper<T> {
 
-    Logger logger = Logger.getLogger("com.github.eyefloaters.console.api.errors.client");
+    Logger logger = Logger.getLogger(getClass());
```

Comment:
I don't have a strong opinion on this, but will point out: Using `getClass()` will mean that for subclasses the line numbers in the logs won't correlate with the apparent log class that appears in the logs. That can be confusing, both for users and maintainers of the project. 

FWIW Strimzi chose the pattern of `static final Logger LOGGER = Logger.getLogger(AbstractNotFoundExceptionHandler.class)`. That doesn't mean you have to do the same.

Reply:
I generally would prefer using the parent class for the parent logger, but in this case since the child classes are empty I think it's more helpful to have their names in the log. This should be revisited if anything changes with this class hierarchy in the future.